spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -1822,8 +1829,13 @@ def go_to_next_todo(self):
                 break
         line_number = block.blockNumber()+1
         self.go_to_line(line_number)
-        self.show_calltip(_("To do"), data.todo,
-                          color='#3096FC', at_line=line_number)
+        self.show_tooltip(
+            _("To do"),
+            data.todo,
+            color='#3096FC',
+            at_line=line_number,
+        )

```

Comment:
These TODO's should behave in the same way as code analysis warnings.

Reply:
See https://github.com/spyder-ide/spyder/pull/9140#discussion_r275781064

Reply:
Ok, thanks for the explanation.